android/KMEA/app/src/main/java/com/tavultesoft/kmea/KMManager.java

```diff
@@ -2075,5 +2091,65 @@ private void keyDownUp(int keyEventCode) {
       IMService.getCurrentInputConnection().sendKeyEvent(new KeyEvent(KeyEvent.ACTION_DOWN, keyEventCode));
       IMService.getCurrentInputConnection().sendKeyEvent(new KeyEvent(KeyEvent.ACTION_UP, keyEventCode));
     }
+
+    /*
+    // TODO: Chromium has a bug where deleteSurroundingText deletes an entire grapheme cluster
+    // instead of one code-point.
```

Comment:
Please add a reference to the bug.
```suggestion
    // instead of one code-point. https://bugs.chromium.org/p/chromium/issues/detail?id=1024738
```